packages/ui/react-ui-editor/src/extensions/comments.ts

```diff
@@ -608,26 +610,17 @@ export const createExternalCommentSync = (
     },
   );
 
-export const useCommentState = (): [{ comment: boolean; selection: boolean }, Extension] => {
-  const [state, setState] = useState<{ comment: boolean; selection: boolean }>({
-    comment: false,
-    selection: false,
-  });
-
-  const observer = useMemo(
+export const useCommentState = (state: ReactiveObject<EditorToolbarState>): Extension => {
```

Comment:
given the changes to how the editor state is being managed it would be good to verify the e2e tests still pass on composer before landing this

Reply:
For sure